wrappers/python/rgb_node.py

```diff
@@ -0,0 +1,62 @@
+"""
+RGB Node Python Wrapper
+"""
+
+import sys
+sys.path.insert(1, '../../bindings/python')
+from rgb import *
+
+class RGBNode:
+    def __init__(self, network, datadir, electrum, verbosity):
+        self.network = network or "testnet"
+        self.datadir = datadir
+        self.electrum = electrum
```

Comment:
It will be good to add default value here like I did in https://github.com/LNP-BP/rgb-sdk/blob/master/wrappers/ecmascript/rgb.js#L7-L11

Reply:
Yes right..